docs/repositories-configure/configuring-code-patterns.md

```diff
@@ -103,19 +90,25 @@ Codacy will use the updated configurations on the next analysis.
     docs/repositories-configure/codacy-configuration-file.md (list of tool short names to use on the Codacy configuration file)
 -->
 
-Codacy supports configuration files for several tools. To use a configuration file for your static analysis tool:
+!!! note
+    -   After activating a configuration file for a tool, Codacy uses that configuration file even if you exclude it from Codacy analysis [using the Codacy UI](ignoring-files.md) or a [Codacy configuration file](codacy-configuration-file.md#syntax-for-ignoring-files).
```

Comment:
We don't need a bullet here, as it's a single note. (just noticed this while giving a last look at the preview)

```suggestion
    After activating a configuration file for a tool, Codacy uses that configuration file even if you exclude it from Codacy analysis [using the Codacy UI](ignoring-files.md) or a [Codacy configuration file](codacy-configuration-file.md#syntax-for-ignoring-files).
```